src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -355,7 +352,10 @@ class SourcesTree extends Component<Props, State> {
   }
 }
 
-function getSourceForTree(state: OuterState, source: ?Source): ?Source | null {
+function getSourceForTree(
+  state: SourcesState,
```

Comment:
small nit, but i think it is nicer to describe it as `AppState` ->  `state: AppState,`